ngraph/test/frontend/shared/src/telemetry.cpp

```diff
@@ -0,0 +1,74 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "telemetry.hpp"
+
+#include <frontend_manager/extension.hpp>
+
+#include "utils.hpp"
+
+using namespace ngraph;
+using namespace ov::frontend;
+
+std::string FrontEndTelemetryTest::getTestCaseName(const testing::TestParamInfo<TelemetryFEParam>& obj) {
+    std::string res = obj.param.m_frontEndName + "_" + obj.param.m_modelName;
+    return FrontEndTestUtils::fileToTestName(res);
+}
+
+void FrontEndTelemetryTest::SetUp() {
+    FrontEndTestUtils::setupTestEnv();
+    m_fem = FrontEndManager();  // re-initialize after setting up environment
+    initParamTest();
+}
+
+void FrontEndTelemetryTest::initParamTest() {
+    m_param = GetParam();
+    m_param.m_modelName = FrontEndTestUtils::make_model_path(m_param.m_modelsPath + m_param.m_modelName);
+}
+
+///////////////////////////////////////////////////////////////////
+
+TEST_P(FrontEndTelemetryTest, testSetElementType) {
```

Comment:
Can you please check the content of "op_count" event here?

Reply:
todo: rename testSetElementType

Reply:
fixed